### DISCUSSION TITLE
Should the Grant Round Info page show a list of contributions to the Round?

### DISCUSSION THREAD

**Discussion by apbendi** [dcgtc/dgrants#191] [Architecture & Design] [answered]
The Grant Round Info page has general info about the nature of the round being run:

![image](https://user-images.githubusercontent.com/1479802/132732281-557140d3-4a53-4400-8691-bdc95c3a6347.png)

Anyone can make contributions to the round's matching pool. Should this page also include a list of contributions to the round? I think we probably would want this. My thinking is the design can look similar/identical to the contributions section of the Grant Detail page, without the middle section:

<img width="1087" alt="Screen Shot 2021-09-09 at 1 17 31 PM" src="https://user-images.githubusercontent.com/1479802/132732514-42b42891-420a-4a60-a550-8ee73e2bd925.png">

**Comment by scco** [comment]
would this use the cart ? so "add round to cart" kind of ?   
or would this  Add Funds trigger a direct metamask popup where you can send X whatever tokens ?

**Reply by apbendi** [reply]
Not through the cart. Yeah, it goes to a simple form where you put in the tokens and then send them via wallet confirmation. See #189 for the WIP branch here from @thelostone-mc. My question is about showing those contributions on the Round page.

**Comment by mds1** [comment, marked as answer]
I'd say in general I agree that it would be nice to show this, but I don't think it needs to be a requirement for the POC

**Reply by apbendi** [reply]
Agreed should be lower priority than plenty of other tasks.

**Reply by phutchins** [reply]
I think it would be nice to see the last donations and where they are going for a grant round. So yes, I think we could show the last X donations with an option to see more later. I agree that this is lower priority than most other things and doesn't need to be a part of the POC if we cannot fit it in time-wise.
